### DISCUSSION TITLE
".NET MAUI Community Toolkit" channel on Discord

### DISCUSSION THREAD

**Discussion by GeorgeLeithead** [CommunityToolkit/Maui#574] [General]
On the discord server `DotNetEvolution` there is a channel for "xamarin-community-toolkit".  Given that there is an ever increasing difference between the XCT and the MCT, would it be prodent to have a new channel created specifically for the "maui-community-toolkit"?

**Comment by VladislavAntonyuk** [comment]
XCT is supported. but it is not popular. So you can ask questions related to maui in that channel as well. in 2 years xct channel will be renamed to mct

**Reply by bijington** [reply]
I think @pictos was trying to see if we could get a channel created for MCT

**Reply by pictos** [reply]
Yes, I'm looking on that

**Comment by CliffAgius** [comment]
Closed as answered
